trl/trainer/online_dpo_trainer.py

```diff
@@ -107,6 +107,10 @@ def __init__(
         optimizers: Tuple[torch.optim.Optimizer, torch.optim.lr_scheduler.LambdaLR] = (None, None),
         preprocess_logits_for_metrics: Optional[Callable[[torch.Tensor, torch.Tensor], torch.Tensor]] = None,
     ) -> None:
+
+        if ref_policy is policy:
```

Comment:
I think we want this here:

```suggestion
        if ref_model is model:
```